components/Card.js

```diff
@@ -32,9 +32,12 @@ const CardContainer = styled.li`
 `;
 
 const DeleteButton = styled.button`
-  background-color: pink;
-  width: 2em;
-  height: 2.2em;
+  background-color: white;
```

Comment:
Browsers interpret values like `white` differently. Please always use Hex, RGB or HSL values

Reply:
Look at your code and correct everything in CSS related to the color values :)

Reply:
> Look at your code and correct everything in CSS related to the color values :)

I will change it to hex codes 👍